api/src/main/java/javax/enterprise/inject/spi/Annotated.java

```diff
@@ -94,6 +113,8 @@
     /**
      * <p>
      * Determine if the program element has an annotation of a certain annotation type.
+     *
+     * If the <tt>annotationType</tt> is repeatable, then returns true if at least one instance is available
```

Comment:
I think we should follow the Reflection API here, i.e. ignore repeatable annotations and return `getAnnotation(annotationClass) != null`.